GameData/RealismOverhaul/RO_SuggestedMods/Buran-Energia/RO_Buran.cfg

```diff
@@ -0,0 +1,492 @@
+@PART[BURAN_CENTER1_MANIPULATOR]
+{
+	%RSSROConfig = True
+	
+	%tags = energia buran vulkan
+}
+
+@PART[BURAN_CENTER2_MANIPULATOR]
+{
+	%RSSROConfig = True
+	
+	%tags = energia buran vulkan
+}
+
+@PART[BURAN_END_MANIPULATOR]
+{
+	%RSSROConfig = True
+	
+	%tags = energia buran vulkan
+}
+
+@PART[BURAN_ROTATOR_MANIPULATOR]
+{
+	%RSSROConfig = True
+	
+	%tags = energia buran vulkan
+}
+
+@PART[BURAN_WRIST_END]
+{
+	%RSSROConfig = True
+	
+	%tags = energia buran vulkan
+}
+
+@PART[BURAN_WRIST_MANIPULATOR]
+{
+	%RSSROConfig = True
+	
+	%tags = energia buran vulkan
+}
+
+@PART[BURAN_CABINE]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 2.0
+	@fuelCrossFeed = True
+	@mass = 3.88
+	
+	%tags = energia buran vulkan
+	
+	!RESOURCE[MonoPropellant]{}
+	
+	!RESOURCE[ElectricCharge]{}
+	
+	@MODULE[ModuleCommand]
+	{
+		@minimumCrew = 1
+	}
+	
+	@MODULE[ModuleRCSFX]
+	{
+		@thrusterPower = 2.9
+		!resourceName = DELETE
+		PROPELLANT
+		{
+			name = Syntin
+			ratio = 0.35652
+		}
+		PROPELLANT
+		{
+			name = Oxygen
+			ratio = 128.696
+		}
+        
+		@atmosphereCurve
+		{
+			@key,0 = 0 295 
+			@key,1 = 1 275
+		}
+	}
+	
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 3300.0
+		type = ServiceModule
+		basemass = -1
+		
+		TANK
+		{
+			name = ElectricCharge
+			amount = 10000.0
+			maxAmount = 10000.0
+		}
+		
+		TANK
+		{
+			name = Food
+			amount = 229.8
+			maxAmount = 229.8
+		}
+		
+		TANK
+		{
+			name = Water
+			amount = 151.9
+			maxAmount = 151.9
+		}
+		
+		TANK
+		{
+			name = Oxygen
+			amount = 342134.0
+			maxAmount = 342134.0
+		}
+		
+		TANK
+		{
+			name = CarbonDioxide
+			amount = 0
+			maxAmount = 13265.9
+		}
+		
+		TANK
+		{
+			name = Waste
+			amount = 0
+			maxAmount = 29.8
+		}
+		
+		TANK
+		{
+			name = WasteWater
+			amount = 0
+			maxAmount = 151.9
+		}
+		
+		TANK
+		{
+			name = Syntin
+			amount = 949.6
+			maxAmount = 949.6
+		}
+		

```

Comment:
Most of these resources should only exist if using TACLS or Kerbalism, so should have a NEEDS block